content/en/templates/taxonomy-templates.md

```diff
@@ -137,9 +141,13 @@ Taxonomies can be ordered by either alphabetical key or by the number of content
 
 ```
 <ul>
-  {{ $data := .Data }}
+  {{ $type := .Type }}
   {{ range $key, $value := .Data.Terms.ByCount }}
-  <li><a href="{{ $.Site.LanguagePrefix }}/{{ $data.Plural }}/{{ $value.Name | urlize }}"> {{ $value.Name }} </a> {{ $value.Count }} </li>
+  {{ $name := .Name }}
```

Comment:
Indent everything under range by 4 spaces

Reply:
Thanks! But it appears that the original indentation is 2-space?